soh/soh/Enhancements/presets.h

```diff
@@ -803,32 +803,32 @@ const std::vector<PresetEntry> spockRacePresetEntries = {
     PRESET_ENTRY_S32("gRandomizeCompleteMaskQuest", 1),
     PRESET_ENTRY_S32("gRandomizeCuccosToReturn", 1),
     PRESET_ENTRY_S32("gRandomizeDampeHint", 1),
-    PRESET_ENTRY_S32("gRandomizeDoorOfTime", 2),
+    PRESET_ENTRY_S32("gRandomizeDoorOfTime", RO_DOOROFTIME_OPEN),
     PRESET_ENTRY_S32("gRandomizeEnableBombchuDrops", 1),
     PRESET_ENTRY_STRING("gRandomizeExcludedLocations", "78,142,143,228,"),
```

Comment:
Not a blocker since we want to move away from this string value for locations, but maybe would be nice to have comment for now indicating what the values are, since I imagine if a new check is added, it would change the meaning of these values.

Reply:
Yea agreed, I wasn't exactly sure how to handle these but can imagine theyre subject to changing a fair bit.  Will try and sort something out later